### ISSUE TITLE
Cannot open/view documents uploaded into project documents

### ISSUE THREAD

**Issue by marypleasant** [ecsion/eSeaGrant-Client-Bug-Tracker#212]
When logged in as a research coordinator, under Projects, I cannot view or open documents that have been uploaded under the Project Documents tab for a given project.

http://eseagrant.oregonstate.edu/index.php?module=Projects&relatedModule=OtherDocs&view=Detail&record=672&mode=showRelatedList&tab_label=Project%20Documents
![pic 1](https://cloud.githubusercontent.com/assets/17168500/14901891/9ad9c566-0d4c-11e6-8d44-2e2a9a9bc469.JPG)
http://eseagrant.oregonstate.edu/index.php?module=OtherDocs&view=Edit&record=673&parentModule=Projects&parent_module=Projects&parent_id=672&tab_label=
![pic 2](https://cloud.githubusercontent.com/assets/17168500/14901896/a3965700-0d4c-11e6-96b1-3ec153464eef.JPG)

**Comment by kdez** [comment]
Yes, I posted about this too - see Issue #206

**Comment by marypleasant** [comment]
It seems like a similar problem, but not the same. We use "Project Documents" tab for documents other than the full proposals, like old progress reports, signed no cost extension, relevant email communications, etc. I would like to be able to see those documents where we uploaded them under Project Documents. When I uploaded documents under this tab (using the "Add Document" function), their title appears, but there is no way to actually open them (that I can find).
